.github/workflows/cd.yml

```diff
@@ -0,0 +1,25 @@
+name: CD
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release:
+    if: "!contains(github.event.head_commit.message, 'skip ci')"
```

Comment:
This is required due to https://github.community/t/github-actions-does-not-respect-skip-ci/17325.

Github actually implemented an [auto generated token](https://docs.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token#using-the-github_token-in-a-workflow) to prevent recursive runs, expect that this token cannot side step branch protections, so users are still forced to bypass with a PAT and this band-aid conditional today if they want to automate protected branches.